modules/compute/gke-node-pool/variables.tf

```diff
@@ -293,3 +293,13 @@ variable "specific_reservation" {
     values = null
   }
 }
+
+variable "compact_placement_policy" {
```

Comment:
Can we instead make placement policy generic and allow customer to pass in which type of placement they would like ? 
Today the only option is "COMPACT" based on https://cloud.google.com/kubernetes-engine/docs/how-to/compact-placement#create_a_compact_placement_policy_in_a_new_cluster

However there might be more options coming.